src/core/src/bootstrap/Constants.py

```diff
@@ -98,6 +98,33 @@ class ConfigSettings(EnumBackport):
     # patch versions for healthstore when there is no maintenance run id
     PATCH_VERSION_UNKNOWN = "UNKNOWN"
 
+    # Strings used in perf logs
+    class LogStrings:
+        TASK = "task"
+        TASK_STATUS = "task_status"
+        ASSESSMENT = "Assessment"
+        PATCH_INSTALLATION = "PatchInstallation"
+        PACKAGE_MANAGER = "package_manager"
+        NUMBER_OF_TRIALS = "number_of_trials"
+        NUMBER_OF_ROUNDS = "number_of_rounds"
+        FAILED = "failed"
+        SUCCEEDED = "succeeded"
+        ERROR_MSG = "error_msg"
+        INSTALLED_UPDATE_COUNT = "installed_update_count"
+        UPDATE_RUN_SUCCESSFUL = "update_run_successful"
```

Comment:
I'm not fully convinced on adding all of these field names in here. Will think a bit more on this but in the meantime, a point to note, as Koshy mentioned in his comment, there have been some naming changes between v1 and v2. "Update" -> "patch" and "run" -> "operation"

Reply:
Done.